### PR TITLE
update the README with some of the things I bumped into

### PR DESCRIPTION
**Pull request by martinbonnin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,20 +8,21 @@
...
+  * activate "Google" as a sign-in provider for Authentication
 
 2. Create a `.env.local` file by copying `.env` file at root folder and set firebase environment variables.
 
-  * REACT_APP_API_KEY=<API_KEY>
-  * REACT_APP_AUTH_DOMAIN=<AUTH_DOMAIN>
-  * REACT_APP_PROJECT_ID=<PROJECT_ID>
+  * REACT_APP_API_KEY=<API_KEY> (settings -> Web API key)
+  * REACT_APP_AUTH_DOMAIN=<AUTH_DOMAIN> (authentication -> Sign-in methods -> Authorized domains)
+  * REACT_APP_PROJECT_ID=<PROJECT_ID> (settings -> project ID)
```
Comment (bpetetot) [inline README.md]: You don't have to go in the settings to get these data.
You can get them from your Firebase project home page:
- Click on "Add an application"
- Select the "Web" icon
And you get all information you need for the app configuration.

Maybe we you add this process in the documentation.

**Review by bpetetot — COMMENTED** [review]
Hi Martin, Thanks for this PR 🙏
I made just a little comment about how to get all project information variables from firebase. Can you change this and then I will merge it 😉

If you are interested to use the app to organize an event or a meetup, I can get you a beta access key for the main platform: https://conference-hall.io
You just have to send me a private message through twitter https://twitter.com/bpetetot with the event, meetup name or your organization.

File: README.md (hunk 1 above)
Comment (martinbonnin) [reply README.md]: done

**Comment by martinbonnin** [comment]
@bpetetot Thanks for the project, it looks and feels very nice :muscle: ! No need for a beta key right now, I was mainly investigating. And I could bootstrap an instance in a breeze anyways :+1: . I updated the readme, let me know what you think.

Also, do you have a place where you discuss development ? I might be able to help on some features if needed ?

**Review by bpetetot — APPROVED** [review]
👍

**Comment by bpetetot** [comment]
Thanks for the PR

Today there is 2 way to discuss about features and development:
1. through GitHub issues
2. there is a "conference-hall" channel on the NantesJS slack http://nantesjs.slack.com

**Pull request merged**